react/features/notifications/reducer.js

```diff
@@ -0,0 +1,43 @@
+import { ReducerRegistry } from '../base/redux';
+
+import {
+    HIDE_NOTIFICATION,
+    SHOW_NOTIFICATION
+} from './actionTypes';
+
+/**
+ * The initial state of the feature notifications.
+ *
+ * @type {array}
+ */
+const DEFAULT_STATE = [];
+
+/**
+ * Reduces redux actions which affect the display of notifications.
+ *
+ * @param {Object} state - The current redux state.
+ * @param {Object} action - The redux action to reduce.
+ * @returns {Object} The next redux state which is the result of reducing the
+ * specified {@code action}.
+ */
+ReducerRegistry.register('features/notifications',
+    (state = DEFAULT_STATE, action) => {
+        switch (action.type) {
+        case HIDE_NOTIFICATION:
+            return state.filter(
+                notification => notification.uid !== action.uid);
+
```

Comment:
It seems inconsistent but I see the new line more often than not in reducer.js files.